apps/nowcasting-app/components/map/delta-color-guide-bar.tsx

```diff
@@ -0,0 +1,74 @@
+import { useMemo } from "react";
+import { ActiveUnit } from "./types";
+
+type DeltaColorGuideBarProps = { unit: ActiveUnit };
+
+const DeltaColorGuideBar: React.FC<DeltaColorGuideBarProps> = ({ unit }) => {
+  const values = useMemo(() => {
+    if (unit === ActiveUnit.MW) {
+      return [
+        { value: "-80", background: "100", opacity: 3, textColor: "black" },
+        { value: "-60", background: "200", opacity: 20, textColor: "black" },
+        { value: "-40", background: "300", opacity: 40, textColor: "ocf-gray-300" },
+        { value: "-20", background: "400", opacity: 60, textColor: "ocf-gray-300" },
+        { value: "0", background: "500", opacity: 80, textColor: "ocf-gray-300" },
+        { value: "+20", background: "600", opacity: 100, textColor: "ocf-gray-300" },
+        { value: "+40", background: "700", opacity: 100, textColor: "ocf-gray-300" },
+        { value: "+60", background: "800", opacity: 100, textColor: "black" },
+        { value: "+80", background: "900", opacity: 100, textColor: "ocf-gray-300" }
+      ];
+    } else if (unit === ActiveUnit.capacity) {
+      return [
+        { value: "-80", background: "100", opacity: 3, textColor: "ocf-gray-300" },
+        { value: "-60", background: "200", opacity: 20, textColor: "ocf-gray-300" },
+        { value: "-40", background: "300", opacity: 40, textColor: "ocf-gray-300" },
+        { value: "300-400", background: "400", opacity: 60, textColor: "black" },
```

Comment:
What's going on here, is this half changed on purpose? Not sure is showing anyway but code looks odd 😂

Reply:
I think I stopped working on this sense we're not doing capacity or percent. Will clean up code.